console/console-log-logged.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
So I am not sure we want to add a real test as part of this PR? I suspect we might need an infrastructure test instead?

Reply:
Renamed and moved the test to `infrastructure/webdriver/bidi/subscription.html`.